packaging/debian-sid/README.Source

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+The packaging is maintained in the upstream git repo at
+
+   github.com/snapcore/snapd:packaging/debian-sid
```

Comment:
This reads like a branch name, is that so? Or is this a directory in the git tree.

Reply:
It's a directory I guess I should make this clearer